app/Commands/Dev/Asm/Compile.hs

```diff
@@ -35,6 +35,8 @@ runCommand opts = do
       TargetWasm32Wasi -> Backend.TargetCWasm32Wasi
       TargetNative64 -> Backend.TargetCNative64
       TargetGeb -> error "GEB target not supported for JuvixAsm"
+      TargetCore -> error "JuvixCore target not supported for JuvixAsm"
```

Comment:
I don't think this has anything to do with autocompletion.

I'm just using a single datastructure for all "compile" commands, because the code for them is almost identical -- to avoid duplication. That's why for 'asm' you have targets that don't make sense.

One could try to remove those targets from the CLI for "asm" or parameterise the data structure by the targets somehow. I judged it not worth the effort since "asm" is an internal command anyway. There are no spurious targets for the main "compile" command.

Do you think it's important enough to spend time on this?

Reply:
Maybe I just give a proper Juvix error here instead of crashing? That's easy to do. Would that be fine?

Reply:
The error is fine, but this should be done:
> One could try to remove those targets from the CLI for "asm" or parameterise the data structure by the targets somehow.

Reply:
if it is done in another pr is also fine

Reply:
Let's open an issue for this and do it in a separate PR. It would be nice, but I don't see this as priority and it requires reworking the module that implements the general "compile" command.